src/averageAges.js

```diff
@@ -15,6 +15,26 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  let manAvarageAge = people.filter((man) => {
+    return man.sex === 'm';
+  });
+
+  if (century) {
+    manAvarageAge = manAvarageAge.filter(man => {
+      return Math.ceil(man.died / 100) === century;
+    });
+  }
+
+  const age = manAvarageAge.map(man => {
+    return man.died - man.born;
+  });
+  const avarageAge = age.reduce((man, prev) => {
+    return man + prev;
+  });
```

Comment:
you did this logic 2 times for 2 different arrays
let's create separate function that will accept an array of people and will return the average age value